po/it.po

```diff
@@ -170,78 +170,80 @@ msgstr "Interfaccia MPRIS2 D-Bus"
 msgid "Add support for MPRIS property LoopStatus"
-msgstr ""
+msgstr "Aggiungi il supporto per la proprietà MPRIS LoopStatus"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:52
 msgid ""
 "Add option --mpv-options for setting arbitrary mpv options from the command-"
 "line"
-msgstr ""
+msgstr "Aggiungi opzione --mpv-options per impostare opzioni mpv arbitrarie dalla linea di-"
+"comando"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:57
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:85
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:113
 msgid "This listing is incomplete. See git log for complete changelog."
-msgstr ""
+msgstr "Questo elenco è incompleto. Vedi log git per il log delle modifiche completo."
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:68
 msgid "Set default screenshot directory to XDG_PICTURES_DIR"
-msgstr ""
+msgstr "Imposta la directory di screenshot predefinita su XDG_PICTURES_DIR"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:71
 msgid ""
 "Improve handling of --window-scale, --autofit, --autofit-larger, and "
 "--autofit-smaller"
-msgstr ""
+msgstr "Migliora la gestione di --scala della finestra, --adattamento automatico, --autoadattamento-largezza e "
+"--autoadattamento-dimensione"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:75
 msgid "Add command line option for setting WM_ROLE"
-msgstr ""
+msgstr "Aggiungi l'opzione della riga di comando per l'impostazione di WM_ROLE"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:78
 msgid "Add context menu item for removing playlist items"
-msgstr ""
+msgstr "Aggiungi la voce del menu contestuale per rimuovere le voci della playlist"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:81
 msgid "Add context menu item for copying location of playlist items"
-msgstr ""
+msgstr "Aggiungi la voce del menu contestuale per copiare la posizione degli elementi della playlist"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:96
 msgid ""
 "Add option to make skip buttons change playlist entries rather than chapters"
-msgstr ""
+msgstr "Aggiungi un'opzione per fare in modo che i pulsanti di salto cambino le voci della playlist anziché i capitoli"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:100
 msgid "Make the file chooser accept non-local locations"
-msgstr ""
+msgstr "Fai in modo che il selettore file accetti posizioni non locali"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:103
 msgid "Add right-click menu entry for looping a single file"
-msgstr ""
+msgstr "Aggiungi la voce di menu del tasto destro del mouse per eseguire il looping di un singolo file"
 
 #: data/io.github.celluloid_player.Celluloid.appdata.xml.in:106
 msgid "Handle property change events for fullscreen and window-scale"
-msgstr ""
+msgstr "Gestire gli eventi di modifica delle proprietà per lo schermo intero e la scala delle finestre"
```

Comment:
Property names (fullscreen and window-scale) should not be translated.

```suggestion
msgstr "Gestire gli eventi di modifica delle proprietà per fullscreen e window-scale"
```

Reply:
I daresay these terms should be put in quotes in English text to avoid ambiguity.